tests/test_integration.py

```diff
@@ -1385,6 +1385,42 @@ def has_doctest1() -> None:
     """
 
 
+Unformatted = TypeVar("Unformatted")
+
+
+@warns("cannot cache unpickable configuration value: 'typehints_formatter'")
```

Comment:
Feels like we should fix this warning not ignore it 🤔

Reply:
#457 is an open issue for this.